client/src/containers/dashboard/Dashboard.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from "react";
```

Comment:
It doesn't look like the commented lines are being used. They may come into use later, but when making PRs, try and have the code as clean as possible